### PR TITLE
Make MIDI output polyphonic

### PR DESCRIPTION
**Pull request by frostburn**
Make it possible to select MIDI input channels

refs #73

### CODE REVIEW

File: src/views/MidiView.vue [hunk 1]
```diff
@@ -75,6 +104,18 @@ onMounted(async () => {
               {{ (input.manufacturer || "(Generic)") + ": " + input.name }}
             </option>
           </select>
+          <div class="control">
```
Comment (frostburn) [inline src/views/MidiView.vue:107]: I have no idea what I'm doing with this CSS. I would like to have the labels under the check boxes like we have in SW1.

Comment (SeanArchibald) [reply src/views/MidiView.vue:107]: There is a div element which wraps around the 16 span elements for the input channels checkboxes. If you give that an id such as input-channels-wrapper or something, then you can add this css:

```
#input-channels-wrapper {
    column-gap: 1rem;
}
#input-channels-wrapper span {
    display: flex;
    flex-flow: column;
    text-align: center;
}
```

I think that will make the check boxes appear the way you want.

Screenshot_20220702_142452

**Comment by frostburn** [comment]
Fixed the channel checkbox layout.

**Comment by frostburn** [comment]
Approved on Discord.

**Pull request merged**